### PR TITLE
GSREN3D-298: fix arrow pointing to ground

### PR DESCRIPTION
**Pull request by ismailsunni**
### JIRA issue

https://jira.camptocamp.com/browse/GSREN3D-298

### Description

- Use the height of the station for the start and end of the arrow. When we have the offset for the arrow, it should be look better

### Screenshots

image

### CODE REVIEW

File: src/helpers/traveltimesHelper.ts [hunk 1]
```diff
@@ -19,11 +21,31 @@ export async function lineStringsFromTraveltimes(
       traveltime.end
     )
 
-    const lineString = new LineString([
-      startTrambusStop?.getGeometry()?.getCoordinates(),
-      endTrambusStop?.getGeometry()?.getCoordinates(),
-    ])
-    return lineString
+    if (is3D) {
+      const cesiumMap = rennesApp.get3DMap()
```
Comment (toni-garcia-camptocamp) [inline src/helpers/traveltimesHelper.ts:25]: Maybe add a comment to explain the code below
For example : "Modify coordinates by adding an altitude" , something like that

File: src/helpers/traveltimesHelper.ts [hunk 2]
```diff
@@ -19,11 +21,31 @@ export async function lineStringsFromTraveltimes(
...
+      const proj3857 = new Projection({
+        epsg: 'EPSG:3857',
+        proj4:
+          '+proj=merc +a=6378137 +b=6378137 +lat_ts=0 +lon_0=0 +x_0=0 +y_0=0 +k=1 +units=m +nadgrids=@null +wktext +no_defs +type=crs',
```
Comment (toni-garcia-camptocamp) [inline src/helpers/traveltimesHelper.ts:29]: This is an interesting part. 
Where did you found the documentation for write the proj4 ? 
I searched on map-core and map-ui but don't found something .

Maybe you have to add a little comment for explain the building the string proj4. Or add a link to the documentation used.

**Review by toni-garcia-camptocamp — COMMENTED** [review]
Work fine on my local, nothing wrong with the code.
Maybe just add both comment , especially for the construction of proj4 which is not easy to understand
But great job !

File: src/helpers/traveltimesHelper.ts (hunk 2 above)
Comment (ismailsunni) [reply src/helpers/traveltimesHelper.ts]: You can get it from https://epsg.io/3857 in the proj.4 section. 

You can also get it from https://spatialreference.org and search for 3857 -> https://spatialreference.org/ref/sr-org/7483/

I follow the example from https://github.com/virtualcitySYSTEMS/map-core/blob/main/src/util/projection.js#L111 and explained in here https://github.com/virtualcitySYSTEMS/map-core/blob/main/src/util/projection.js#L10

Anyway, after I read again that source code, I switch to using the existing `mercatorProjection`

**Pull request merged**